.teamcity/src/subprojects/build/core/ProjectCore.kt

```diff
@@ -8,6 +8,12 @@ data class OSEntry(val name: String, val agentString: String, val taskName: Stri
 data class JavaScriptEngine(val name: String, val dockerContainer: String)
 data class OSJDKEntry(val osEntry: OSEntry, val jdkEntry: JDKEntry)
 
+const val reportArtifacts = """                        
```

Comment:
Should we have a `trimIndent` call here?

Reply:
Yeah.